pkgs/development/libraries/gnutls/generic.nix

```diff
@@ -13,6 +13,8 @@ stdenv.mkDerivation rec {
   inherit src patches;
 
   configureFlags = [
+    # FIXME: perhaps use $SSL_CERT_FILE instead
+    "--with-default-trust-store-file=/etc/ssl/certs/ca-certificates.crt"
```

Comment:
Hmm, I guess the configure flag should only be added on Linux? Without it, configure probes a few options, which might give a correct result, as darwin builds is are not chrooted.